evap/staff/importers.py

```diff
@@ -318,13 +315,13 @@ def process(cls, excel_file, semester, vote_start_date, vote_end_date, test_run)
             importer = cls()
             importer.read_book(excel_file)
             if importer.errors:
-                return importer.success_messages, importer.warnings, importer.errors
+                return importer.success_messages, dict(importer.warnings), importer.errors
```

Comment:
Iterating defaultdicts in a django template does not work (see https://code.djangoproject.com/ticket/16335). You have to convert them to normal dicts before. I could move the conversion to where the dict is passed to the template though. I thought it might be simpler and cleaner the way I did it since this approach creates more encapsulation (The "default"-part of the dict is not needed anywhere outside the importer).